databroker/core.py

```diff
@@ -585,9 +642,12 @@ def __init__(self,
         self._get_event_pages = get_event_pages
         self._get_event_count = get_event_count
         self._get_resource = get_resource
+        self._get_resources = get_resources
         self._lookup_resource_for_datum = lookup_resource_for_datum
         self._get_datum_pages = get_datum_pages
-        self.filler = filler
+        self.fillers = {'yes': event_model.Filler(),
```

Comment:
These will need a handler registry. Also, not sure we want a separate `Filler` instance per `BlueskyRun`. One role of the `Filler` object is the control caching of documents and files. If two users request the same file, would be need to have one file handle.